introduction.md

```diff
@@ -0,0 +1 @@
+Hello. My name is Chris.
```

Comment:
Good point. Here is that update. https://github.com/deadlyvipers/dojo_rules/pull/10117/commits/1ed80f88ac16e4629c1aaa124696286d0569d670